docs/pages/components/SearchField.mdx

```diff
@@ -96,6 +106,12 @@ By default, SearchField is wrapped in a `<form role="search">` to provide better
       description:
         'If true, the label and input will be rendered in a `<form>` tag. If false, the label and input will be rendered in a `<div>` tag.',
       defaultValue: 'true'
+    },
+    {
+      name: 'trailingChildren',
+      type: ['ReactNode, string'],
```

Comment:
Debating between ReactNode vs ReactElement, or vs both. Not 100% sure about the difference.